### ISSUE TITLE
Footer not fixed to bottom of screen

### ISSUE THREAD

**Issue by jesseallhands** [Couchers-org/couchers#2115]
The footer isn't fixed to the bottom of the screen. See example in the screenshot below:

![image](https://user-images.githubusercontent.com/79384268/137826082-bdee6df1-9469-4e3e-8798-eb4888479cea.png)

This appears to be the case on all pages where the footer appears including the following pages:

https://app.couchers.org/ (dashboard)
https://app.couchers.org/contribute
https://app.couchers.org/donate

**Comment by sadiejay** [comment]
@aapeliv I would like to tackle this issue! It's my first time contributing to this codebase and may have some questions. Thank you!

**Comment by lucaslcode** [comment]
Hi @sadiejay! I'll assign you. Feel free to ask me here or email me any questions. 

There are a few ways to approach this and the simpler the better - maybe something like having the main content have a minimum height rather than using a root flex layout.

**Comment by sadiejay** [comment]
Just an update, I'm working on this issue now, because of the holidays and travel I've been a bit slower than anticipated! 

I will be reaching out about my progress soon.

Thanks again!

**Comment by sadiejay** [comment]
Hi everyone, Happy New Year!! 

I'm trying to poke around using Gitpod (avoiding Docker), however I'm having issues recreating the footer in the screenshot.

I've checked Chrome, Safari and Firefox, and on my device the footer is fixed to the bottom of the screen. Any guidance would be appreciated!

**Comment by sadiejay** [comment]
### Firefox
![Screenshot 2022-01-11 at 14-23-08 Couchers org Beta](https://user-images.githubusercontent.com/19538219/149015747-f0ca0d87-fed5-49ae-9876-1b8c51747af2.png)

### Safari
<img width="1392" alt="Screen Shot 2022-01-11 at 2 22 32 PM" src="https://user-images.githubusercontent.com/19538219/149015769-a025c136-01be-4a07-939f-e811534d7b7b.png">

### Chrome
<img width="1275" alt="Screen Shot 2022-01-11 at 2 21 52 PM" src="https://user-images.githubusercontent.com/19538219/149015774-2c8e31b4-4792-4ab3-b3fa-55abc8019cf5.png">

@lucaslcode (forgot to tag you in previous message)

**Comment by lucaslcode** [comment]
Try a short page like messages, and use developer tools to use a taller screen size (you can set zoom to 75 or 50% if you're screen resolution isn't tall enough)

**Comment by sadiejay** [comment]
Thank you for the tip @lucaslcode! I was able to recreate the issue using your advice.

As of yesterday, I'm noticing now that the frontend isn't building on gitpod — I believe that it's something to do with:

> Also, looks like there's some compile error from the existing frontend tests because of the proto contract change in the `accountInfo` query
https://github.com/Couchers-org/couchers/pull/2397#issuecomment-1015976009

How should I proceed?  I don't have any experience with docker, but I'm willing to try it out if that's a solution!

**Comment by aapeliv** [comment]
@sadiejay: thanks for pointing that out, I think I fixed that bug!

I think you can get developing here: https://gitpod.io/#https://github.com/Couchers-org/couchers/pull/2402

**Comment by aapeliv** [comment]
@sadiejay: that's been merged and fixed now! Give it another shot :)

**Comment by sadiejay** [comment]
@aapeliv Everything is checking out now thanks! :)

**Comment by sadiejay** [comment]
Hi @lucaslcode ! I'm stuck and wondering how to move forward.

I've tried different flexbox-free solutions with varying success but I'm running into the issue of mobile devices in landscape mode having content obscured or other pages with content overlapping.

How should I proceed? Thank you!

Messages page on mobile device working correctly:

![Screen Shot 2022-02-17 at 13 19 25](https://user-images.githubusercontent.com/19538219/154555284-5c5414f6-78ec-4dc2-af5b-0bac9f264b4e.png)

![Screen Shot 2022-02-17 at 13 19 08](https://user-images.githubusercontent.com/19538219/154555200-41540db8-f55d-47fc-8fe0-983c2839532a.png)

Donate pages on mobile device with content overlapping:

![Screen Shot 2022-02-17 at 13 19 58](https://user-images.githubusercontent.com/19538219/154555230-4d2ff6da-f9db-4af5-b07e-9ec8ec1f598b.png)
